### PR TITLE
trt-589 library-go bump

### PR DESCRIPTION
**Pull request by neisw**
[Trt-589](https://issues.redhat.com//browse/Trt-589) is researching an issue where installer pods timeout, shutdown and enter a failed state triggering KubePodNotReady alerts. library-go was updated to add milestone logging to track the progress and track down the issue. Bumping the version to pull those changes in.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,39 +6,39 @@ require (
...
 	github.com/vishvananda/netlink v1.0.0
-	go.etcd.io/etcd/api/v3 v3.5.0
-	go.etcd.io/etcd/client/pkg/v3 v3.5.0
-	go.etcd.io/etcd/client/v3 v3.5.0
-	go.etcd.io/etcd/server/v3 v3.5.0
+	go.etcd.io/etcd/api/v3 v3.5.4
+	go.etcd.io/etcd/client/pkg/v3 v3.5.4
+	go.etcd.io/etcd/client/v3 v3.5.4
+	go.etcd.io/etcd/server/v3 v3.5.4
 	go.etcd.io/etcd/tests/v3 v3.5.0
```
Comment (deads2k) [inline go.mod:28]: update this one.

Comment (deads2k) [inline go.mod:28]: this version is wrong

File: pkg/operator/etcd_assets/bindata.go [hunk 2]
```diff
@@ -1287,11 +1287,13 @@ var _bindata = map[string]func() (*asset, error){
 // directory embedded in the file by go-bindata.
 // For example if you run go-bindata on data/... and data contains the
 // following hierarchy:
-//     data/
-//       foo.txt
-//       img/
-//         a.png
-//         b.png
+//
```
Comment (tjungblu) [inline pkg/operator/etcd_assets/bindata.go:1290]: that's unfortunately a difference from another go version. Did you run `make update` with go 1.18? The #939 has the same issue bumping to 1.19.

TBH I'm actually fine with you bumping the go.mod and build version to 1.19 here too so we can close the ART PR.

Comment (tjungblu) [reply pkg/operator/etcd_assets/bindata.go:1290]: awesome, thanks!

**Comment by tjungblu** [comment]
building seems currently not working somehow, but 
/lgtm

**Pull request merged**